DTAdmin/Models/DataManager.swift

```diff
@@ -166,6 +166,85 @@ class DataManager: HTTPManager {
         }
     }
     
+    func getListOfQuestions(forEntity typeEntity: Entities, entityId: String, quantity:UInt, fromNo index: UInt, completionHandler: @escaping (_ listEntity: [Any]?, _ error: String?) -> ()) {
+        let indexString = String(index)
+        let quantityString = String(quantity)
+        guard let request = self.getURLReqest(entityStructure: typeEntity, type: TypeReqest.GetRecordsRangeByTest, id: entityId, limit: quantityString, offset: indexString) else {
+            let error = "Cannot prepare header for URLRequest"
+            completionHandler(nil, error)
+            return
+        }
+        self.getResponse(request: request) { (list, error) in
+            if let error = error {
+                StoreHelper.logout()
```

Comment:
Why do we need this line?

Reply:
I think, if response has an error we want to logout from the program